04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+In addition to the above suggestions, area limited foraging where agents would only be allowed to forage with only one neighbouring island using a ranking algorithm. This could also be enforced and implemented in a future iteration. Moreover, this particular scenario could be extended by programming prey densities to be geofocused, where prey populations could vary between regions on the “map”. Islands would then need to make a strategic plan on how their ranking of neighbouring islands would change in accordance to their returns and returns broadcasted from other island pairs.\\
+
+Finally, more foraging options could be added in future implementations. For example, a third foraging technique, call it whale it hunting, could be created. That would have an even higher return than the deer hunting, deeming it an appealing forage method. The barriers of entry could be similar to that of the deer hunting method in this scenario but at the same time, there would be a more spaced out/expensive tier system. This combination would potentially incentivise cooperation between islands, if their strategy was to maximise returns. Also, as we have introduced more foraging methods, it could be the case that we hard-fix the type of return for each method and observe how agents decide to behave. For example, we could have the deer and fish hunting methods result in returns proportional to inputs, whereas the whale hunting method would result in an equal split of returns between all participating islands. Would this mean that islands choose a fairer division of returns and not go on whale foraging at all? Or Would they be intrigued by the higher costs and choose whale hunting over the other foraging methods? In the latter question agents would need to take in mind that an island may be entering an insignificant amount reaping however, a large reward. It is therefore up to the islands to decide if the higher payoff is worth the risk of other islands free-riding.\\
+
+All of the aforementioned behaviours are of course influenced by island tactics and therefore, an island programmed to be greedy will not cease to be greedy no matter how the surrounding islands’ behaviours change, unless the greedy island can adapt. Therefore, conclusions drawn on specific foraging methods, and how agents react with them, need to take this into account.\\
```

Comment:
good sentence -- could you keep the same style of this sentence through the whole document :D 

BUT the comma is used wrongly in the last sentence (as written in the guidelines)..

should be:
conclusions **drawn on specific foraging methods and how agents react with them need** to take this into account.

Reply:
Suggested rewrite: 

All the aforementioned behaviours are, of course, influenced by island tactics. Therefore, an island programmed to be greedy will not cease to be greedy, no matter how the surrounding islands’ behaviours change, unless the greedy island can adapt. Therefore, agent strategies must attempt to draw conclusions on specific foraging methods and modify their strategies.